src/test/kotlin/org/rust/lang/core/psi/RsDebuggerExpressionCodeFragmentCompletionTest.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.psi
+
+import org.rust.ProjectDescriptor
+import org.rust.WithStdlibRustProjectDescriptor
+
+@ProjectDescriptor(WithStdlibRustProjectDescriptor::class)
```

Comment:
That's reasonable, I tried to do that. But for some reason, other tests don't work without the descriptor. When removing the top-level descriptor, I get `Expected completions that contain ..., but no completions found` assertion for all other tests in this class

Reply:
It's because you have only one item to complete. In this case, `myFixture.completeBasic()` returns `null` and insert item automatically into editor. Check `myFixture.file.text` before `checkNotNull` call in `org.rust.lang.core.completion.RsCompletionTestFixtureBase#doContainsCompletion`
I see two possible solutions:
- avoid single completion
- use `org.rust.lang.core.completion.RsCompletionTestFixtureBase#doSingleCompletion`

Reply:
Oh, thanks! Indeed, I have faced this problem several times before when working on code completion, and each time took quite a lot of time to remember that fact and solve the problem. Now I'm wondering maybe we could check the API or something to make it easier to use?

Reply:
Looks like I fixed this behaviour: https://github.com/intellij-rust/intellij-rust/pull/8070/commits/38e66fdf2976947b2c615b13cb5895808a73cb79